packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -147,6 +151,20 @@ class InspectorTreeControllerFlutter extends Object
     }
     return _maxIndent;
   }
+
+  bool get isDebugLayoutSummaryEnabled => _isDebugLayoutSummaryEnabled;
```

Comment:
any suggestion on the variable name is appreciated, I'm still unsure about this naming :sweat_smile: